src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  const baseCost = days * dailyRate;
+  let discount = 0;
+
+  if (days >= 7) {
+    discount = 50;
+  } else if (days >= 3) {
+    discount = 20;
+  }
```

Comment:
Remove the 'else' keyword to simplify the control flow, as per the checklist items 4 and 5. Since the 'if' block contains a 'return', the 'else' is not necessary.